src/gameplayv2/resistance/tests/component.test.js

```diff
@@ -0,0 +1,20 @@
+import { Resistance } from '../entities/resistance'
+
+test('properly initialize the Spy entity', () =>{
+    //do test case here..
+});
+
+test('properly initialize the Resistance entity', () =>{
+    //do test case here..
+    //example:
+    expect(1+2).toBe(3);
+
+    //or for an array:
+    const arraychar = ['a', 'b']
+    const array = [1, 2]
+    //this will throw an Error
+    expect(cloneArray(array)).toEqual(arraychar);
+});
+
+//take a look at rewards/tests/getRewards.test.ts
+//if you want to run this, do npm test on terminal
```

Comment:
Nit: Usually we have a space after the `//`. Prettier (auto-formatter) should handle this for you! I'd recommend installing it if you haven't already!